### ISSUE TITLE
Error `attempt to call field 'nvim__set_hl_ns' (a nil value)` in Nightly

### ISSUE THREAD

**Issue by datwaft** [NarutoXY/dim.lua#15]
The following error appears after updating to the latest version of Neovim nightly:

```
Error in packer_compiled: ...al/share/nvim/site/pack/packer/start/dim.lua/lua/dim.lua:158: attempt to call field 'nvim__set_hl_ns' (a nil value)
```

This was probably caused by https://github.com/neovim/neovim/issues/14090#issuecomment-1218115927, which is a breaking change.

**Comment by NarutoXY** [comment]
Thanks for reporting this issue :)

But unfortunately I've exams rn and also am sick as hell so it may take some time to fix this issue. I hope you understand the situation.

**Comment by datwaft** [comment]
No problem! Your health is priority.

Meanwhile, here are some workarounds for anyone also having this issue:

- Locking Neovim to the commit before https://github.com/neovim/neovim/commit/35653e6bcda6923b5213fb9356c35067d6d0288f.
- Removing the plugin from your configuration temporarily while waiting for the fix.

I chose the second one as it is easier than having to compile Neovim from source.

Take care!

**Comment by Gelio** [comment]
I attempted to fix the plugin in #16. It occurred to me that calling `nvim_set_hl_ns` is not necessary by this plugin. It is possibly even harmful since another plugin could also call `nvim_set_hl_ns` and cause this plugin to stop working. I made it work without changing the global highlight namespace
